Mathlib/Algebra/BigOperators/Multiset/Basic.lean

```diff
@@ -153,6 +153,10 @@ theorem pow_count [DecidableEq α] (a : α) : a ^ s.count a = (s.filter (Eq a)).
 #align multiset.pow_count Multiset.pow_count
 #align multiset.nsmul_count Multiset.nsmul_count
 
+theorem filter_prod_mul_filter_neg (p) [DecidablePred p] :
```

Comment:
Finset lemma is called [Finset.prod_filter_mul_prod_filter_not](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/BigOperators/Basic.html#Finset.prod_filter_mul_prod_filter_not)

Reply:
Thanks!